src/node/channels.h

```diff
@@ -23,6 +23,13 @@ namespace ccf
   using SeqNo = uint64_t;
   using GcmHdr = crypto::GcmHeader<sizeof(SeqNo)>;
 
+#ifndef NDEBUG
+  static constexpr size_t message_limit = 2048;
```

Comment:
Perhaps add a comment here to explain why we do this (i.e. see a few refresh during our perf test)?